### PR TITLE
config changes for pay-hub call

### PR DESCRIPTION
**Pull request by mdayican**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-765

### Change description ###

config changes to call payhub 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/bulk-scan-payment-processor/values.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ java:
   ingressHost: bulk-scan-payment-processor-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   environment:
     PAYMENTS_QUEUE_MAX_DELIVERY_COUNT: "5"
+    PAY_HUB_URL:  https://ccpay-bulkscanning-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```
Comment (nitinprabhu) [inline charts/bulk-scan-payment-processor/values.yaml]: https won't work I think

Comment (mdayican) [reply charts/bulk-scan-payment-processor/values.yaml]: fixed

**Comment by mdayican** [comment]
> override `PAY_HUB_URL` in preview yaml to use AAT one

added

File: charts/bulk-scan-payment-processor/values.preview.template.yaml [hunk 2]
```diff
@@ -5,6 +5,7 @@ java:
       key: connectionString
   environment:
     PAYMENTS_QUEUE_READ_CONNECTION_STRING: "$(SB_CONN_STRING);EntityPath=payments"
+    PAY_HUB_URL: "https://ccpay-bulkscanning-api-aat.service.core-compute-aat.internal"
```
Comment (rkondratowicz) [inline charts/bulk-scan-payment-processor/values.preview.template.yaml]: https or http?

Comment (mdayican) [reply charts/bulk-scan-payment-processor/values.preview.template.yaml]: I think in preview https.

Comment (mdayican) [reply charts/bulk-scan-payment-processor/values.preview.template.yaml]: > https or http?

made http.

**Comment by mdayican** [comment]
> the chart version bump has gone :(

updated

**Pull request merged**